internal/admission-webhook-server.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 The Lokomotive Authors
```

Comment:
Perhaps `internal/admission-webhook-server/admission-webhook-server.go` would be a better place for this code.